### PR TITLE
docs: Add MAC VTAP example

### PR DESCRIPTION
**Pull request by marituhone**
Enhancement: Improved Configuration for MAC VTAP

Reason: The current setup process for MAC VTAP lacks a standardized and reusable approach,Currently we are supporting MAC VTAP as "tap": True in macvlan. To address this, this pull request introduces a yml  file, "macvtap.yml", to provide a  structured method for configuring MAC VTAP. This enhancement aims to simplify the setup process and improve the usability of MAC VTAP.

Result: With the addition of the YAML "macvtap.yml" file, users can easily define and modify MAC VTAP settings by modifying this file. This will streamline the configuration process and enable users to customize MAC VTAP according to their specific requirements. 

Issue Tracker Tickets (Jira or BZ if any):  issue #285

### CODE REVIEW

File: examples/macvtap.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Manage network MAC VLAN
```
Comment (ffmancera) [inline examples/macvtap.yml:3]: It should be `Manage network MAC VTAP` I think

**Review by ffmancera — CHANGES_REQUESTED** [review]

**Comment by liangwen12year** [comment]
`PR Title lint` is failed, you can take a look at https://linux-system-roles.github.io/contribute.html, specifically the section about `Write a good PR title and description`. In short, you can change the title into `docs: Add MAC VTAP example`.

**Comment by liangwen12year** [comment]
Can you squash the PR into only one commit with the commit title `docs: Add MAC VTAP example`.

**Comment by spetrosi** [comment]
> Can you squash the PR into only one commit with the commit title `docs: Add MAC VTAP example`.

fyi, we moved the requirement to indicate change type to the PR title only. Commits now can be in any form. Though I like to still add a `type:` prefix to the commit for clarity.

**Comment by liangwen12year** [comment]
https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec, please look at this guide on how to squash the commits.

**Comment by marituhone** [comment]
@liangwen12year  i did squash them

**Comment by liangwen12year** [comment]
> @liangwen12year i squashed them

Can you check why the pr also has the commit - `Merge branch 'linux-system-roles:main' into issueFix` ?

**Comment by marituhone** [comment]
@liangwen12year  i fix it

**Review by liangwen12year — APPROVED** [review]

**Comment by liangwen12year** [comment]
> @liangwen12year i have fixed it

@marituhone thanks for contributing to it.

**Comment by liangwen12year** [comment]
Nit-picking, can you change the commit title to `docs: Add MAC VTAP example`?

**Review by richm — APPROVED** [review]

**Pull request merged**